desktop/renderer/index.html

```diff
@@ -9,16 +9,20 @@
   <body>
     <div id="app"><div title="loading..." style="flex: 1;background-color: #f5f5f5"></div>
     <script>
+      const ipcRenderer = require('electron').ipcRenderer
+      const util = require('util')
       function logToElectron() {
         ['log', 'warn', 'error'].forEach(key => {
           window.console[`_${key}`] = window.console[key]
-          window.console[key] = function() {
-            window.console[`_${key}`](...arguments)
-            require('electron').ipcRenderer.send('console.' + key, Array.prototype.slice.call(arguments))
+          window.console[key] = (...args) => {
+            window.console[`_${key}`](...args)
+            const toSend = util.inspect(args)
```

Comment:
this fix will be removed in this branch: https://github.com/keybase/client/pull/4566. there's another case where this can happen which i'm fixing in a new commit now